distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Dialect.enso

```diff
@@ -364,7 +364,9 @@ agg_count_distinct args = if args.is_empty then (Error.throw (Illegal_Argument.E
 ## PRIVATE
 agg_count_distinct_include_null args =
     ## If we always count as tuples, then even null fields are counted.
-    SQL_Builder.code "COUNT(DISTINCT (" ++ SQL_Builder.join ", " args ++ ", 0))"
+    ## But Snowflake seems to not like tuples?
+    #SQL_Builder.code "COUNT(DISTINCT (" ++ SQL_Builder.join ", " args ++ ", 0))"
```

Comment:
Is there a TODO to restore this previous verison?

Reply:
I'm pretty sure we will need to fix handling it properly to make our tests pass. So I think no TODO shall be needed.